projects/igniteui-angular/src/lib/core/styles/components/splitter/_splitter-theme.scss

```diff
@@ -78,9 +79,15 @@
     $hitbox-debug-color: rgba(coral, .24);
 
     $variant: map.get($theme, '_meta', 'variant');
-    $size: map-get($theme, 'size');
-    $splitter-size: ($size / ($size * 0 + 1) + 2);
-    $slim-splitter: calc( 1 / $splitter-size);
+
+    //remove unit from size
+    $size: math.div(map-get($theme, 'size'), map-get($theme, 'size') * 0 + 1);
```

Comment:
Why do you multiply the size * 0 and then add 1 to it just to arrive at size = size / 1 which is simply the value of size? s / s * 0 * 1 = s?

Reply:
Yeah, I was amazed myself but obviously this is the way to remove the unit of the variable :D Dividing it just by 1 doesn't work. I don't know why, honostly

Reply:
Ok, got it. Can you make this as a function in the theming library, under the utils/math module?